### ISSUE TITLE
[suggest]make pino system tray and toolbar icon changable

### ISSUE THREAD

**Issue by GoogleCodeExporter** [KGIII/pino-twitter#154]
```
like other applications, if icon themes provide the icons named like "pino-
tray-icon", the pino tray icon can change to use this icon.

and the toolbar icon can use the icons already exist in icon themes like 
"user-home.png".
```

Original issue reported on code.google.com by `jarry...@gmail.com` on 27 Mar 2010 at 12:20

**Comment by GoogleCodeExporter** [comment]
Original comment by `tro...@gmail.com` on 27 Mar 2010 at 12:29
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Closed by f1d12d856b revision.
```

Original comment by `tro...@gmail.com` on 11 Apr 2010 at 2:20
- Changed state: **Fixed**
